client/app/templates/components/project-milestone.hbs

```diff
@@ -26,9 +26,11 @@
       </sup>
     {{/if}}
   </h4>
+
   {{#if milestone.displayDate}}
```

Comment:
It looks like `{{#if milestone.displayDate2}}` is within the `{{#if (in-array (array "In Progress" "Completed") milestone.statuscode)}}`

Should `{{#if milestone.displayDate}}` also be within that if?

Reply:
I don't think so. It looks like the outer  `{{#if milestone.displayDate}}` is used to conditionally display milestone links and outcomes. The new requirement to add `{{#if (in-array (array "In Progress" "Completed") milestone.statuscode)}}` only mentioned it is for conditionally displaying/hiding the dates, not the milestone links and outcomes